### ISSUE TITLE
0323_2モデル評価

### ISSUE THREAD

**Issue by smith-217** [smith-217/Nishika#4]
- 特徴量：
['市区町村コード',
 '市区町村名',
 '地区名',
 '最寄駅：名称',
 '最寄駅：距離（分）',
 '間取り',
 '面積（㎡）',
 '建物の構造',
 '都市計画',
 '建ぺい率（％）',
 '容積率（％）',
 '改装',
 '西暦',
 '築年数',
 '用途_その他',
 '用途_事務所',
 '用途_事務所、店舗',
 '用途_住宅',
 '用途_住宅、その他',
 '用途_住宅、店舗',
 '用途_住宅、駐車場',
 '用途_倉庫',
 '用途_店舗',
 '用途_駐車場',
 '今後の利用目的_その他',
 '今後の利用目的_事務所',
 '今後の利用目的_住宅',
 '今後の利用目的_店舗',
 '取引の事情等_その他事情有り',
 '取引の事情等_他の権利・負担付き、調停・競売等',
 '取引の事情等_瑕疵有りの可能性',
 '取引の事情等_調停・競売等',
 '取引の事情等_関係者間取引',
 'bert_pca2_1',
 'bert_pca2_2',
 'bert_pca3_1',
 'bert_pca3_2',
 'bert_pca3_3',
 'w2v_pca2_1',
 'w2v_pca2_2',
 'w2v_pca3_1',
 'w2v_pca3_2',
 'w2v_pca3_3',
 'bert_svd2_1',
 'bert_svd2_2',
 'bert_svd3_1',
 'bert_svd3_2',
 'bert_svd3_3',
 'w2v_svd2_1',
 'w2v_svd2_2',
 'w2v_svd3_1',
 'w2v_svd3_2',
 'w2v_svd3_3',
 '取引時点_年',
 '取引時点_四半期']
- モデル：
  - LGBM
  - KFold
  - parameter:
SEED = 0
N_SPLITS = 5
SHUFFLE = True
lgb_params = {'num_leaves': 32,
               'min_data_in_leaf': 64,
               'objective': 'mae',
               'max_depth': -1,
               'learning_rate': 0.01,
               "boosting": "gbdt",
               "bagging_freq": 1,
               "bagging_fraction": 0.8,
               "bagging_seed": SEED,
               "verbosity": -1,
              'reg_alpha': 0.1,
              'reg_lambda': 0.3,
              'colsample_bytree': 0.4,
              'metric':"mae",
              'num_threads':6,
               'num_boost_round': 10000,
               'early_stopping_rounds': 500,
               'verbose_eval': 200,
               }
- スコア：
  - CV：0.07645404587070907
  - LB：0.0771
